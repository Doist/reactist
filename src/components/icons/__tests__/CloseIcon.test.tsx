import React from 'react'
import { render } from '@testing-library/react'

import CloseIcon from '../CloseIcon.svg'

describe('CloseIcon', () => {
    it('renders a svg icon', () => {
        const { container } = render(<CloseIcon />)
        expect(container).toMatchSnapshot()
    })
})
