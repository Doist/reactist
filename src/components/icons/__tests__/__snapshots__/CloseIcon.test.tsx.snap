// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon renders a svg icon 1`] = `
<div>
  <svg
    height="24"
    viewBox="0 0 24 24"
    width="24"
  >
    <path
      d="M11.293 12L5.146 5.854a.5.5 0 1 1 .708-.708L12 11.293l6.146-6.147a.5.5 0 0 1 .708.708L12.707 12l6.147 6.146a.5.5 0 0 1-.708.708L12 12.707l-6.146 6.147a.5.5 0 0 1-.708-.708L11.293 12z"
      fill="gray"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
